functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/ScheduledJobSpec.groovy

```diff
@@ -17,41 +14,24 @@ class ScheduledJobSpec extends SeleniumBase{
      * Schedules a job and checks if with time passing, executions reflects in activity section.
      *
      */
-    def "Scheduled job"(){
+    def "scheduled job"(){
         given:
         def projectName = "scheduled-job-test"
         setupProject(projectName)
         LoginPage loginPage = page LoginPage
         ActivityPage activityPage = page ActivityPage
-        JobCreatePage jobCreatePage = page JobCreatePage
         JobListPage jobListPage = page(JobListPage)
         jobListPage.loadJobListForProject(projectName)
+        def jobXml = JobUtils.generateScheduledJobsXml("scheduledJob", "<time hour='*' seconds='*/5' minute='*' />")
```

Comment:
Just to comment. this was a great idea, using the API instead of the GUI reduce allot of steps that can fail randomly.